src/style.css

```
@@ -1,3 +1,76 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+:root {
+  --active: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 0 50px;
+  margin: auto;
+
+  @media screen and (min-width: 1024px) {
+    max-width: 1024px;
+  }
+
+  @media screen and (min-width: 1200px) {
+    max-width: 1200px;
+  }
+}
+
+.logo {
+  margin: 0;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0 -20px 0 0;
```

```suggestion
  margin: 0;
}

 .nav__item:last-child {
    margin: 0;
 }
   
```
we can just remove the margin from the last child

---

+